## Problem Statement - Normal Distribution

The mean weight of a morning breakfast cereal pack is 0.295 kg with a standard deviation of 0.025 kg. The random variable weight of the pack follows a normal distribution. 

  a)What is the probability that the pack weighs less than 0.280 kg?
  
  b)What is the probability that the pack weighs more than 0.350 kg? 
  
  c)What is the probability that the pack weighs between 0.260 kg to 0.340 kg? 

In [1]:
import scipy.stats as stats

Incase of normal distribtion standard normal variable is defined as Z = (x-mean)/standard deviation

In [2]:

z=(0.28-0.295)/0.025
print(z)

-0.5999999999999983


In [3]:
z
"""when z value is negative, answer is direct but when it is +ve, then the answer is 1-z"""

'when z value is negative, answer is direct but when it is +ve, then the answer is 1-z'

In [4]:
"""To calculate the probability that the pack weighs less than 0.280 kg"""
print("the probability that the pack weighs less than 0.280 kg is:", stats.norm.cdf(-0.6))


the probability that the pack weighs less than 0.280 kg is: 0.2742531177500736


In [5]:
z=(0.35-0.295)/0.025

In [6]:
z

2.1999999999999997

In [7]:
print("the probability that the pack weighs more than 0.350 kgs is: ", 1 - stats.norm.cdf(2.2))

the probability that the pack weighs more than 0.350 kgs is:  0.01390344751349859


In [8]:
z1= (0.26-0.295)/ 0.025
z1

-1.399999999999999

In [9]:
z2 = (0.36-0.295)/0.025
z2

2.6

In [10]:
z=stats.norm.cdf(z2) - stats.norm.cdf(z1)
z

0.9145821527425101

In [11]:
stats.norm.cdf(0.91)

0.8185887451082028

In [12]:
print("the probability that the pack weighs between 0.260 kg to 0.340 kg is: ",  1 - stats.norm.cdf(0.91))

the probability that the pack weighs between 0.260 kg to 0.340 kg is:  0.1814112548917972


that means there is 18% probability that pack weighs between 0.260 kg to 0.340 kg 